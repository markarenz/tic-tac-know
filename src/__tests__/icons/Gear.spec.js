import { render, screen } from '@testing-library/react';
import Gear from '../../icons/Gear';

describe('Gear', () => {
  it('renders component', () => {
    const container = render(<Gear />);
    expect(container).toMatchSnapshot();
  });
});
