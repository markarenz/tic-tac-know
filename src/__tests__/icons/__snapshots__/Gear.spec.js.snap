// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Gear renders component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <svg
        viewBox="0 0 46 46"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M41.737 18.842h-2.231a16.585 16.585 0 00-1.906-4.517l1.604-1.608c.433-.433.67-1.01.67-1.624 0-.612-.239-1.19-.67-1.623L36.53 6.8c-.867-.869-2.382-.868-3.246-.002L31.614 8.46a16.606 16.606 0 00-4.456-1.81V4.264c0-1.266-1.012-2.307-2.278-2.307h-3.779c-1.266 0-2.262 1.04-2.262 2.307V6.65a16.651 16.651 0 00-4.466 1.81l-1.67-1.662c-.866-.868-2.382-.867-3.251 0L6.78 9.467a2.308 2.308 0 00-.672 1.623c0 .613.237 1.19.67 1.623l1.604 1.611a16.583 16.583 0 00-1.905 4.517H4.245a2.282 2.282 0 00-2.29 2.281V24.9c0 1.267 1.023 2.262 2.29 2.262h2.232a16.665 16.665 0 001.904 4.526l-1.604 1.614a2.284 2.284 0 00-.672 1.626c0 .614.239 1.193.672 1.626l2.673 2.673c.434.433 1.01.67 1.622.67.613 0 1.19-.238 1.623-.67l1.678-1.663c1.372.8 2.85 1.415 4.466 1.811v2.384c0 1.266.996 2.287 2.262 2.287h3.78a2.275 2.275 0 002.277-2.287v-2.385a16.615 16.615 0 004.457-1.81l1.664 1.659c.434.435 1.01.673 1.625.673.613 0 1.19-.239 1.624-.671l2.673-2.67c.433-.433.67-1.01.672-1.623 0-.614-.24-1.19-.672-1.624l-1.602-1.62a16.733 16.733 0 001.904-4.526h2.232c1.266 0 2.307-.995 2.307-2.262v-3.777c.001-1.263-1.039-2.281-2.305-2.281zM22.99 30.2c-4.008 0-7.258-3.22-7.258-7.189 0-3.972 3.25-7.187 7.258-7.187 4.009 0 7.26 3.215 7.26 7.187 0 3.97-3.251 7.189-7.26 7.189z"
          fill="#ccc"
        />
        <path
          d="M21.1 1c-1.79 0-3.216 1.495-3.216 3.263v1.754c-1.156.34-2.247.783-3.276 1.327L13.38 6.122l-.004-.001c-1.26-1.26-3.338-1.258-4.599 0L6.106 8.792a3.248 3.248 0 00-.002 4.598l-.004-.002 1.16 1.166a17.494 17.494 0 00-1.404 3.331h-1.61C2.467 17.885 1 19.345 1 21.124V24.9c0 1.782 1.48 3.217 3.247 3.217h1.608a17.57 17.57 0 001.405 3.344L6.1 32.63A3.257 3.257 0 006.1 37.23l2.674 2.674h.001a3.252 3.252 0 004.594 0v.001l1.235-1.225c1.03.545 2.123.989 3.28 1.329v1.75c0 1.768 1.437 3.243 3.216 3.243h3.78c1.78 0 3.233-1.467 3.233-3.242v-1.753a17.55 17.55 0 003.269-1.327l1.223 1.22a3.253 3.253 0 004.598.002l2.674-2.672a3.26 3.26 0 00.953-2.296v-.002c0-.861-.345-1.688-.95-2.296l-.002-.002-1.158-1.17a17.634 17.634 0 001.407-3.347h1.607c1.767 0 3.263-1.426 3.263-3.217v-3.776c.002-1.789-1.482-3.239-3.26-3.239h-1.608a17.497 17.497 0 00-1.408-3.331l1.16-1.16a3.255 3.255 0 000-4.6l-2.674-2.67c-1.26-1.262-3.341-1.262-4.599 0l-1.226 1.22a17.543 17.543 0 00-3.269-1.327V4.263c0-1.776-1.443-3.264-3.233-3.264zm0 1.913h3.78c.742 0 1.322.593 1.322 1.35V7.4l.73.179a15.63 15.63 0 014.201 1.706l.633.37 2.194-2.182.002-.001c.468-.47 1.42-.472 1.893.001l2.672 2.672c.255.256.391.585.391.948 0 .366-.133.691-.39.947l-2.135 2.14.39.641a15.633 15.633 0 011.797 4.258l.184.717h2.972c.754 0 1.35.586 1.35 1.325V24.9c0 .744-.586 1.306-1.352 1.306h-2.971l-.186.719a15.776 15.776 0 01-1.796 4.266l-.386.639 2.128 2.151v.002c.258.259.393.584.393.948a1.316 1.316 0 01-.39.947l-2.674 2.67a1.316 1.316 0 01-.948.392c-.367 0-.691-.133-.95-.392v-.002l-2.184-2.178-.635.37a15.638 15.638 0 01-4.204 1.708l-.73.177v3.137c0 .757-.568 1.33-1.32 1.33H21.1c-.753 0-1.306-.565-1.306-1.33v-3.135l-.728-.179a15.74 15.74 0 01-4.212-1.708l-.633-.37-2.2 2.18-.001.002a1.316 1.316 0 01-.95.392c-.365 0-.687-.134-.946-.392l-2.673-2.672a1.316 1.316 0 01-.391-.95c0-.365.135-.694.391-.949l.002-.002 2.131-2.146-.388-.64a15.712 15.712 0 01-1.793-4.266l-.184-.72H4.246c-.766 0-1.334-.552-1.334-1.305v-3.776a1.31 1.31 0 011.334-1.327H7.22l.184-.719c.39-1.523 1-2.95 1.795-4.256l.39-.64-2.135-2.144a1.304 1.304 0 01-.392-.946c0-.347.149-.703.394-.948l2.671-2.67c.475-.473 1.427-.473 1.9 0l2.193 2.182.635-.37a15.675 15.675 0 014.21-1.708l.73-.177V4.263c0-.765.563-1.35 1.305-1.35zm1.89 11.956c-4.519 0-8.214 3.65-8.214 8.143 0 4.49 3.694 8.145 8.214 8.145 4.52 0 8.215-3.653 8.216-8.145 0-4.493-3.696-8.143-8.216-8.143zm0 1.911c3.498 0 6.305 2.782 6.305 6.232 0 3.448-2.807 6.234-6.305 6.234-3.496 0-6.303-2.787-6.303-6.234 0-3.45 2.807-6.232 6.303-6.232z"
          fill="#000"
        />
      </svg>
    </div>
  </body>,
  "container": <div>
    <svg
      viewBox="0 0 46 46"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M41.737 18.842h-2.231a16.585 16.585 0 00-1.906-4.517l1.604-1.608c.433-.433.67-1.01.67-1.624 0-.612-.239-1.19-.67-1.623L36.53 6.8c-.867-.869-2.382-.868-3.246-.002L31.614 8.46a16.606 16.606 0 00-4.456-1.81V4.264c0-1.266-1.012-2.307-2.278-2.307h-3.779c-1.266 0-2.262 1.04-2.262 2.307V6.65a16.651 16.651 0 00-4.466 1.81l-1.67-1.662c-.866-.868-2.382-.867-3.251 0L6.78 9.467a2.308 2.308 0 00-.672 1.623c0 .613.237 1.19.67 1.623l1.604 1.611a16.583 16.583 0 00-1.905 4.517H4.245a2.282 2.282 0 00-2.29 2.281V24.9c0 1.267 1.023 2.262 2.29 2.262h2.232a16.665 16.665 0 001.904 4.526l-1.604 1.614a2.284 2.284 0 00-.672 1.626c0 .614.239 1.193.672 1.626l2.673 2.673c.434.433 1.01.67 1.622.67.613 0 1.19-.238 1.623-.67l1.678-1.663c1.372.8 2.85 1.415 4.466 1.811v2.384c0 1.266.996 2.287 2.262 2.287h3.78a2.275 2.275 0 002.277-2.287v-2.385a16.615 16.615 0 004.457-1.81l1.664 1.659c.434.435 1.01.673 1.625.673.613 0 1.19-.239 1.624-.671l2.673-2.67c.433-.433.67-1.01.672-1.623 0-.614-.24-1.19-.672-1.624l-1.602-1.62a16.733 16.733 0 001.904-4.526h2.232c1.266 0 2.307-.995 2.307-2.262v-3.777c.001-1.263-1.039-2.281-2.305-2.281zM22.99 30.2c-4.008 0-7.258-3.22-7.258-7.189 0-3.972 3.25-7.187 7.258-7.187 4.009 0 7.26 3.215 7.26 7.187 0 3.97-3.251 7.189-7.26 7.189z"
        fill="#ccc"
      />
      <path
        d="M21.1 1c-1.79 0-3.216 1.495-3.216 3.263v1.754c-1.156.34-2.247.783-3.276 1.327L13.38 6.122l-.004-.001c-1.26-1.26-3.338-1.258-4.599 0L6.106 8.792a3.248 3.248 0 00-.002 4.598l-.004-.002 1.16 1.166a17.494 17.494 0 00-1.404 3.331h-1.61C2.467 17.885 1 19.345 1 21.124V24.9c0 1.782 1.48 3.217 3.247 3.217h1.608a17.57 17.57 0 001.405 3.344L6.1 32.63A3.257 3.257 0 006.1 37.23l2.674 2.674h.001a3.252 3.252 0 004.594 0v.001l1.235-1.225c1.03.545 2.123.989 3.28 1.329v1.75c0 1.768 1.437 3.243 3.216 3.243h3.78c1.78 0 3.233-1.467 3.233-3.242v-1.753a17.55 17.55 0 003.269-1.327l1.223 1.22a3.253 3.253 0 004.598.002l2.674-2.672a3.26 3.26 0 00.953-2.296v-.002c0-.861-.345-1.688-.95-2.296l-.002-.002-1.158-1.17a17.634 17.634 0 001.407-3.347h1.607c1.767 0 3.263-1.426 3.263-3.217v-3.776c.002-1.789-1.482-3.239-3.26-3.239h-1.608a17.497 17.497 0 00-1.408-3.331l1.16-1.16a3.255 3.255 0 000-4.6l-2.674-2.67c-1.26-1.262-3.341-1.262-4.599 0l-1.226 1.22a17.543 17.543 0 00-3.269-1.327V4.263c0-1.776-1.443-3.264-3.233-3.264zm0 1.913h3.78c.742 0 1.322.593 1.322 1.35V7.4l.73.179a15.63 15.63 0 014.201 1.706l.633.37 2.194-2.182.002-.001c.468-.47 1.42-.472 1.893.001l2.672 2.672c.255.256.391.585.391.948 0 .366-.133.691-.39.947l-2.135 2.14.39.641a15.633 15.633 0 011.797 4.258l.184.717h2.972c.754 0 1.35.586 1.35 1.325V24.9c0 .744-.586 1.306-1.352 1.306h-2.971l-.186.719a15.776 15.776 0 01-1.796 4.266l-.386.639 2.128 2.151v.002c.258.259.393.584.393.948a1.316 1.316 0 01-.39.947l-2.674 2.67a1.316 1.316 0 01-.948.392c-.367 0-.691-.133-.95-.392v-.002l-2.184-2.178-.635.37a15.638 15.638 0 01-4.204 1.708l-.73.177v3.137c0 .757-.568 1.33-1.32 1.33H21.1c-.753 0-1.306-.565-1.306-1.33v-3.135l-.728-.179a15.74 15.74 0 01-4.212-1.708l-.633-.37-2.2 2.18-.001.002a1.316 1.316 0 01-.95.392c-.365 0-.687-.134-.946-.392l-2.673-2.672a1.316 1.316 0 01-.391-.95c0-.365.135-.694.391-.949l.002-.002 2.131-2.146-.388-.64a15.712 15.712 0 01-1.793-4.266l-.184-.72H4.246c-.766 0-1.334-.552-1.334-1.305v-3.776a1.31 1.31 0 011.334-1.327H7.22l.184-.719c.39-1.523 1-2.95 1.795-4.256l.39-.64-2.135-2.144a1.304 1.304 0 01-.392-.946c0-.347.149-.703.394-.948l2.671-2.67c.475-.473 1.427-.473 1.9 0l2.193 2.182.635-.37a15.675 15.675 0 014.21-1.708l.73-.177V4.263c0-.765.563-1.35 1.305-1.35zm1.89 11.956c-4.519 0-8.214 3.65-8.214 8.143 0 4.49 3.694 8.145 8.214 8.145 4.52 0 8.215-3.653 8.216-8.145 0-4.493-3.696-8.143-8.216-8.143zm0 1.911c3.498 0 6.305 2.782 6.305 6.232 0 3.448-2.807 6.234-6.305 6.234-3.496 0-6.303-2.787-6.303-6.234 0-3.45 2.807-6.232 6.303-6.232z"
        fill="#000"
      />
    </svg>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
